Modelica/Fluid/Utilities.mo

```diff
@@ -343,7 +344,7 @@ k1=1, k2=3 is shown in the next figure:</p>
 </dl>
 </html>", revisions="<html>
 <ul>
-<li><em>Sept., 2019</em>
+<li><em>Sept., 2010</em>
     by <a href=\"mailto:Martin.Otter@DLR.de\">Martin Otter</a>:<br>
```

Comment:
Is this on purpose?

Reply:
Yes, as it was wrong in 9f2493d3c9bcd16ed070ec0690ec9925f5ab23f3.